django/db/models/sql/query.py

```diff
@@ -82,7 +82,11 @@ def get_children_from_q(q):
 class RawQuery:
     """A single raw SQL query."""
 
-    def __init__(self, sql, using, params=()):
+    has_select_fields = True
+    contains_aggregate = []
+    contains_over_clause = False
+
+    def __init__(self, sql, using="default", params=()):
```

Comment:
Lets keep this PR focused on ticket-21604; making `using` default to `"default"` should be tackled somewhere else.

```suggestion
    def __init__(self, sql, using, params=()):
```